Tensile/Common.py

```diff
@@ -726,7 +726,7 @@
     # 0:   Disable StoreRemap (default)
     # 1~8: Enable StoreRemap and set the global write vector width
     # Suggest optimum value: fp32 = [2,4], fp16 or bf16 = [4,8] (dwordx2 and dowrdx4)
-    "StoreRemapVectorWidth":      [0,1,2,4,8],
+    "StoreRemapVectorWidth":      [-1,0,1,2,4,8],
```

Comment:
Can you explain this new value in the comment above?

Reply:
I believe this is cherry-picked from other commit, maybe @aazz44ss can answer how he did the auto-assignment for the -1 value?

Reply:
@aazz44ss  Can you please offer some insight on the -1 setting above? Thanks!